railgun/config/crd/bases/configuration.konghq.com_kongclusterplugins.yaml

```diff
@@ -14,6 +14,7 @@ spec:
     listKind: KongClusterPluginList
     plural: kongclusterplugins
     singular: kongclusterplugin
+  preserveUnknownFields: false
```

Comment:
Useful field. Do you have a test case covering this field ?

Reply:
I didn't add a test case, because the field is statically false in v1 of `CustomResourceDefinition` so there's not really anything much to test, however in a later PR as per follow up and the scope of https://github.com/Kong/charts/issues/391 a chart test will be added to validate the upgrade.

Reply:
hmm, a follow up is good. Having been understand this PR will give user a smooth upgrade from version v1beta1 to v1, not aware we need other supplement.

Reply:
Sounds good, let's follow up :+1: